cmd/cli/plugin/airgapped-network/cmd/interface.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cmd ImgPkgClient defines functions to pull/push/List images
+package cmd
+
+type ImgPkgClient interface {
+	ImgpkgCopyImagefromtar(sourceImageName string, destImageRepo string, customImageRepoCertificate string) error
+	ImgpkgCopytotar(sourceImageName string, destImageRepo string) error
```

Comment:
`ImgpkgCopyImagefromtar` => `ImgpkgCopyImageFromTar` and `ImgpkgCopytotar` => `ImgpkgCopyToTar `?